configure.in

```diff
@@ -5489,7 +5500,9 @@ if test "$WXGTK2" = 1; then
     if test "$wxUSE_PRINTING_ARCHITECTURE" = "yes" ; then
 
         if test "$wxUSE_GTKPRINT" = "yes" ; then
-            if test "$WXGTK3" = 1; then
+            if test "$WXGTK4" = 1; then
+                gtk_unix_print="gtk+-unix-print-4.0"
+            elif test "$WXGTK3" = 1; then
```

Comment:
I think this could be simplified as `gtk_unix_print="gtk+-unix-print-$TOOLKIT_VERSION.0"`